### DISCUSSION TITLE
Content examples for application/xml are wrapped in '<notagname>'

### DISCUSSION THREAD

**Discussion by CJKinni** [fastapi/fastapi#8440] [Questions] [answered]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
@router.get("/{app_no}/xml",
    responses={
        200: {
            "description": "Example description",
            "content": {
                "application/xml": { 
                    "example": """
<?xml version="1.0" encoding="UTF-8"?>
<CATALOG>
  <CD>
    <TITLE>Empire Burlesque</TITLE>
    <ARTIST>Bob Dylan</ARTIST>
    <COUNTRY>USA</COUNTRY>
    <COMPANY>Columbia</COMPANY>
    <PRICE>10.90</PRICE>
    <YEAR>1985</YEAR>
  </CD>
</CATALOG>""",
                },
                "application/json": None
            },
        }
    })
```

Produces the following example in the `/docs` UI:

```
<?xml version="1.0" encoding="UTF-8"?>
<notagname>
&lt;?xml version=&quot;1.0&quot; encoding=&quot;UTF-8&quot;?&gt;
&lt;CATALOG&gt;
  &lt;CD&gt;
    &lt;TITLE&gt;Empire Burlesque&lt;/TITLE&gt;
    &lt;ARTIST&gt;Bob Dylan&lt;/ARTIST&gt;
    &lt;COUNTRY&gt;USA&lt;/COUNTRY&gt;
    &lt;COMPANY&gt;Columbia&lt;/COMPANY&gt;
    &lt;PRICE&gt;10.90&lt;/PRICE&gt;
    &lt;YEAR&gt;1985&lt;/YEAR&gt;
  &lt;/CD&gt;
&lt;/CATALOG&gt;</notagname>
```
```


### Description

I'm trying to setup an `application/xml` content example with a hardcoded response.

When I set a response, it ends up wrapped in `notagname` tags and escaped.  What is the accepted way to hardcode an example response for an XML?

### Operating System

macOS

### Operating System Details

_No response_

### FastAPI Version

0.75.0

### Python Version

Python 3.8.9

### Additional Context

_No response_

**Comment by vicente23** [comment]
Hello @CJKinni,

I have the same problem. Have you found the solution since ? 

Thanks

**Comment by LimpingBear** [comment]
Hi.
This isn't working for me either and any help would be appreciated.
I have a simple yaml example I tried in https://editor.swagger.io/ that was also partially generated with the editors insert function
```yaml 
openapi: 3.0.1
info:
  title: test
  version: 1.0.0
servers:
  - url: /
paths:
  /test:
    get:
      operationId: test
      responses:
        '200':
          description: Success
          content:
            text/xml:
              examples:
                test:
                  value: >-
                    <?xml version="1.0" encoding="utf-8"?><test>test</test>
            text/xmml:
              examples:
                test:
                  value: >-
                    <?xml version="1.0" encoding="utf-8"?><test>test</test>
```

With the content type text/xml the <notagname> element is added and with the content type text/xmml (which is wrong) the example works. 

I noticed it the <notagname> element is added for all content types ending with 'xml'

**Comment by mhrimaz** [comment]
This is a problem with with swagger-ui : https://github.com/swagger-api/swagger-ui/blob/1f81a1f2da10d3080e08914c36f9a1b1da984dc4/src/core/plugins/json-schema-2020-12-samples/fn/main.js#L60

because if you use another mime type, it will render properly without escaping.

**Reply by mhrimaz** [reply, marked as answer]
I found something working. you need the schema as it is. I use fastapi 0.104.1.
![image](https://github.com/tiangolo/fastapi/assets/17963017/6a48ae2e-160b-42ba-a8f6-a8519c32d856)

```python
'application/xml': {
    'example': """<conceptDescription><id>something_8ccad77f</id></conceptDescription>""",
    'schema': {'type': 'object', 'format': 'xml' , "xml":{"name":"conceptDescription"}}
}
```
